README.md

```diff
@@ -463,14 +463,16 @@ If we don’t pass timeout to set_log_level(), it will raise a Lua error.
 
 resty.kong.log.get\_log\_level
 ----------------------------------
-**syntax:** *resty.kong.log.get_log_level(level)*
+**syntax:** *current_level, timeout, original_level = resty.kong.log.get_log_level()*
 
 **context:** *any*
 
 **subsystems:** *http*
 
-If the dynamic log level is set, it will return the dynamic log level,
-otherwise it will return `level`.
+Returns the current dynamic log level, remaining timeout and the original log level.
+
+If the dynamic log level is not set or not active,
+`timeout` will be `0`.
```

Comment:
TODO: we may need to clarify that the `timeout` unit is in seconds, and it is an integer (no decimals, no milliseconds)

Reply:
Could you create a doc PR to add it? thanks.

Reply:
fixed in https://github.com/Kong/lua-kong-nginx-module/pull/89, help me review it, ty!